agent/connect/ca/provider_vault.go

```diff
@@ -223,6 +230,12 @@ func (v *VaultProvider) renewToken(ctx context.Context, watcher *vaultapi.Lifeti
 	go watcher.Start()
 	defer watcher.Stop()
 
+	retrier := retry.Waiter{
+		MinFailures: 1,
+		MinWait:     1 * time.Second,
+		Jitter:      retry.NewJitter(20),
+	}
+
```

Comment:
minor: Were these values arrived at empirically? It may be worth adding a short comment indicating why these values are set the way they are. My initial thought was _"do these need to be configurable?"_. The answer is probably _"No."_, but it would be nice to be certain about that :)